x-pack/plugin/watcher/src/internalClusterTest/java/org/elasticsearch/xpack/watcher/actions/webhook/WebhookTokenIntegrationTests.java

```diff
@@ -85,14 +85,18 @@ public void stopWebservice() throws Exception {
         webServer.close();
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/93536")
     public void testWebhook() throws Exception {
+        assumeFalse(
+            "Cannot run in FIPS mode since the keystore will be password protected and sending a password in the reload"
+                + "settings api call, require TLS to be configured for the transport layer",
+            inFipsJvm()
+        );
```

Comment:
This is copied from https://github.com/elastic/elasticsearch/blob/ff870e185897ff13c95309a684131761fa03e986/server/src/internalClusterTest/java/org/elasticsearch/action/admin/ReloadSecureSettingsIT.java#L49-L53